topicExtractor.py

```diff
@@ -152,14 +153,15 @@ def fitTopicModel(self):
                     logging.warning(
                         "All topics are -1. Retrying with K-means clustering..."
                     )
-                    # This import should not occur frequently, so we only call when it is needed.
-                    from sklearn.cluster import KMeans
+                    KMeansAttempt = self.useKMeans(vectorizerModel, docs)
```

Comment:
I feel this `KMeansAttempt` variable can be describe what it is holding. can it be renamed like `KMeanAttemptWithVecModelAndNone`? That a long name.

or rename the variable in L160 can be simple rename `KMeansWithoutModel` and rename variable L156 to `KMeansWithModel`

This suggestion is for code readability purposes.

Reply:
I have added a new variable `KMeansAttemptWithoutVectorizer` to track this for better readability in the latest commit.